doc/releases/release-notes-3.7.rst

```diff
@@ -846,6 +848,7 @@ Drivers and Sensors
     from Power Off state on STM32 L4, U5, WB, & WL SoC series.
   * Added driver for Analog Devices MAX32 SoC series.
   * Added support for Nuvoton Numaker M2L31X series.
+  * Added interrupt support to the Renesas RZ/T2M GPIO driver.
```

Comment:
Could you add a link to the corresponding dt compatible page ```(:dtcompatible:`renesas,rzt2m-gpio`)```

Reply:
Sure, done